### ISSUE TITLE
Do not write empty logs for checkpointed tasks

### ISSUE THREAD

**Issue by annawoodard** [Parsl/parsl#1651]
At the moment, when a checkpointed bash app task is skipped because it has been checkpointed, the log will still be created, but will be empty. This is confusing. I think we should either soft-link to the original log or copy the original log, if it is available.

**Comment by benclifford** [comment]
another option is that the monitoring DB lets you find other executions of "the same" task based on checkpoint identity and you can find logs that way.

**Comment by annawoodard** [comment]
Good point, but I favor having old-fashioned log debugging operate the same way whether or not monitoring is disabled.

**Comment by benclifford** [comment]
that means we have to further duplicate run information (log paths) between the "monitoring" db and the "checkpointing" db; and it means checkpointed tasks that "are not" run end up incurring more IO overhead if logs files are large. The first one concerns me from a consistency perspective; the second from actual workflows that I'm expecting to run at scale.

**Comment by annawoodard** [comment]
Good point. To clarify: I'm open to no log being created, too. I just think empty logs are confusing
